### PR TITLE
Tctl add scenario explanations

### PR DESCRIPTION
**Pull request by DJSanti**
Query and Signal pages now have scenarios written in. This should help a user determine when they'd be using tctl commands.

This is a trial for future page edits. Many of the tctl pages give the syntax, modifiers, and not much else.

### CODE REVIEW

File: docs/tctl/workflow/query.md [hunk 1]
```diff
@@ -8,9 +8,34 @@ tags:
...
+Query result as JSON:
+4
+```
+
+## Modifiers
+
+Queries are written as follows:
+
+```bash
+tctl workflow query [modifiers]
```
Comment (flossypurse) [inline docs/tctl/workflow/query.md:37]: Which are the required modifiers, and which are the optional?

File: docs/concepts/what-is-a-signal.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ A Signal is an external asynchronous request to a [Workflow Execution](/concepts
```
Comment (flossypurse) [inline docs/concepts/what-is-a-signal.md:12]: Let's put these how-to links here
- How to develop Signals (link to app dev guide)
- How to send a Signal using tctl (link to tctl)

(and remove the how-to link at the bottom of this page.

File: docs/tctl/workflow/query.md [hunk 3]
```diff
@@ -8,9 +8,34 @@ tags:
   - tctl
 ---
 
-The `tctl workflow query` command sends a Query to a [Workflow Executions](/concepts/what-is-a-workflow-execution).
+The `tctl workflow query` command sends a Query to a [Workflow Executions](/concepts/what-is-a-workflow-execution). Queries require a running worker process to execute its callback code.
```
Comment (flossypurse) [inline docs/tctl/workflow/query.md:11]:
```suggestion
The `tctl workflow query` command sends a [Query](/concepts/what-is-a-query) to a [Workflow Execution](/concepts/what-is-a-workflow-execution).
```

File: docs/tctl/workflow/signal.md [hunk 4]
```diff
@@ -10,7 +10,80 @@ tags:
 
 The `tctl workflow signal` command [Signals](/concepts/what-is-a-signal) a [Workflow Execution](/concepts/what-is-a-workflow-execution).
 
-`tctl workflow signal [<modifiers>]`
+## Signals with tctl
```
Comment (flossypurse) [inline docs/tctl/workflow/signal.md:13]: Drop this header, not needed.

File: docs/tctl/workflow/query.md (hunk 1 above)
Comment (DJSanti) [reply docs/tctl/workflow/query.md:37]: workflow_id is required for query and signal. the rest are optional

**Review by flossypurse — APPROVED** [review]
This is gtg.
There is an opportunity to re-write this content and make it pop.
But for now, it clears out more from the legacy page - and thats a win.

File: docs/concepts/what-is-a-signal.md [hunk 5]
```diff
@@ -8,15 +8,14 @@ tags:
   - explanation
 ---
 
-A Signal is an asynchronous request to a [Workflow Execution](/concepts/what-is-a-workflow-execution).
+A Signal is an asynchronous request to a [Workflow Execution](/concepts/what-is-a-workflow-execution). Signals can be [defined in code](/app-dev-context/signals)or executed on the [command line](/tctl/workflow/signal).
```
Comment (flossypurse) [inline docs/concepts/what-is-a-signal.md]:
```suggestion
A Signal is an asynchronous request to a [Workflow Execution](/concepts/what-is-a-workflow-execution).

- [How to develop, send, and handle Signals in code](/application-development/features#signals)
- [How to send a Signal using tctl](/tctl/workflow/signal).
```

File: docs/concepts/what-is-a-signal.md [hunk 6]
```diff
@@ -8,15 +8,14 @@ tags:
   - explanation
 ---
 
-A Signal is an asynchronous request to a [Workflow Execution](/concepts/what-is-a-workflow-execution).
+A Signal is an asynchronous request to a [Workflow Execution](/concepts/what-is-a-workflow-execution). Signals can be [defined in code](/application-development/features#signals)or executed on the [command line](/tctl/workflow/signal).
```
Comment (flossypurse) [inline docs/concepts/what-is-a-signal.md:11]:
```suggestion
A Signal is an asynchronous request to a [Workflow Execution](/concepts/what-is-a-workflow-execution).

- [How to develop, send, and handle Signals in code](/application-development/features#signals)
- [How to send a Signal using tctl](/tctl/workflow/signal)
```

**Pull request merged**